src/impact-dashboard/EpidemicModelContext.tsx

```diff
@@ -101,13 +137,34 @@ interface ResetPayload {
   countyName?: string;
 }
 
-function getResetState(payload: ResetPayload): EpidemicModelState {
-  const { dataSource, stateCode = "US Total", countyName = "Total" } = payload;
-  // some defaults can (indeed, must) bet set even without external data
-  const resetBase = {
+function getLocaleData(
+  dataSource: CountyLevelData,
+  stateCode: string,
+  countyName: string,
+) {
+  return {
+    stateCode,
+    countyName,
     countyLevelData: dataSource,
+    countyLevelDataLoading: false,
+    totalIncarcerated: dataSource.get(stateCode)?.get(countyName)
+      ?.totalIncarceratedPopulation,
+    hospitalBeds: dataSource.get(stateCode)?.get(countyName)?.hospitalBeds,
+    ageUnknownPopulation: dataSource.get(stateCode)?.get(countyName)
+      ?.totalIncarceratedPopulation,
+    ageUnknownCases: dataSource.get(stateCode)?.get(countyName)
+      ?.estimatedIncarceratedCases,
+    confirmedCases: dataSource.get(stateCode)?.get(countyName)
+      ?.estimatedIncarceratedCases,
```

Comment:
I know this was already the case, but should this actually be _estimated_ here? The same value as for `ageUnknownCases`?

Reply:
"confirmed cases" may be kind of a misnomer; this is also what the spreadsheet does. we adjust the number of confirmed cases we get from the NY Times dataset by some underreporting factor and use that as the default. it _is_ confusing, maybe we ought to just rename it?

Reply:
Got it, thanks for the context. Let's file that away as a tech debt task. I'd rather not make any modeling changes at this precise instance in time, to minimize regression risk.